lib/plugins/physics.js

```diff
@@ -117,17 +138,24 @@ function inject (bot, { physicsEnabled }) {
     return dYaw
   }
 
-  function updatePosition (dt) {
-    // If you're dead, you're probably on the ground though ...
-    if (!bot.isAlive) bot.entity.onGround = true
+  function isDead () {
```

Comment:
This function should be named something else. It does not return the alive status of the bot but a helper variable to send packets after death.

Reply:
This function returns if the bot is considered "dead" or not to send physics packets. Vanilla client sends movement packets for 20 ticks after dying.

Reply:
Maybe `isNotSendingPosition`